[MINI-5845] Support closing a MiniApp programmatically (#471)
rakutentech/ios-miniapp: USERGUIDE.md

}
```

> <a id="miniapp-close-event-listner></a>

Reviewer: ```suggestion
<a id="miniapp-close></a>
```

Author: Thank you I have updated in the latest commit.

### MiniApp Close

MiniApp can request the host app to close itself through the javascript bridge event trigger.

> MiniApp-SDK provides the `MiniAppMessageDelegate` interface method `func closeMiniApp(withConfirmation: , completionHandler:)` which must be implemented in the host app.

Reviewer: ```suggestion
MiniApp iOS SDK provides the `MiniAppMessageDelegate` interface method `func closeMiniApp(withConfirmation: , completionHandler:)` which can be implemented in the host app.
```

Reviewer: Also, you can mention that this interface is optional

Author: Thank you I have updated in the latest commit.

```swift
extension ViewController: MiniAppMessageDelegate {

>     func closeMiniApp(withConfirmation: Bool, completionHandler: @escaping (Result<Bool, MASDKError>) -> Void) {

Reviewer: ```suggestion
    func closeMiniApp(withConfirmation: Bool, completionHandler: @escaping (Result<Bool, MiniAppJavaScriptError >) -> Void) {
```

Author: Thank you I have updated in the latest commit.
